requirements.txt

```diff
@@ -0,0 +1,24 @@
+argparse
```

Comment:
There's already a requirements.txt in https://github.com/PX4/PX4-Autopilot/blob/main/Tools/setup/requirements.txt

Reply:
Oh, I didn't see that. I'll fix.